bitnami/harbor/values.yaml

```diff
@@ -0,0 +1,654 @@
+  debug: false
+
+## Bitnami Harbor Portal image
+## ref: https://hub.docker.com/r/bitnami/harbor-portal/tags/
+##
+portalImage:
+  registry: gcr.io
+  repository: bitnami-labs/harbor-portal
+  tag: 1.8.0-centos-7
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+  ## Set to true if you would like to see extra information on logs
+  ## It turns BASH and NAMI debugging in minideb
+  ## ref:  https://github.com/bitnami/minideb-extras/#turn-on-bash-debugging
+  debug: false
+
+## Bitnami Harbor Job Service image
+## ref: https://hub.docker.com/r/bitnami/harbor-jobservice/tags/
+##
+jobserviceImage:
+  registry: gcr.io
+  repository: bitnami-labs/harbor-jobservice
+  tag: 1.8.0-centos-7
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+  ## Set to true if you would like to see extra information on logs
+  ## It turns BASH and NAMI debugging in minideb
+  ## ref:  https://github.com/bitnami/minideb-extras/#turn-on-bash-debugging
+  debug: false
+
+## Bitnami Harbor Registry image
+## ref: https://hub.docker.com/r/bitnami/harbor-registry/tags/
+##
+registryImage:
+  registry: gcr.io
+  repository: bitnami-labs/harbor-registry
+  tag: 1.8.0-centos-7
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+  ## Set to true if you would like to see extra information on logs
+  ## It turns BASH and NAMI debugging in minideb
+  ## ref:  https://github.com/bitnami/minideb-extras/#turn-on-bash-debugging
+  debug: false
+
+## Bitnami Harbor Registryctl image
+## ref: https://hub.docker.com/r/bitnami/harbor-registryctl/tags/
+##
+registryctlImage:
+  registry: gcr.io
+  repository: bitnami-labs/harbor-registryctl
+  tag: 1.8.0-centos-7
```

Comment:
Do not use rolling tags. Same with the rest

Reply:
Yes! I will update the image sections once the containers are published in the final destination.